ProblemSolvingMentoringSession/Part 1/AbsoluteDifferenceChallenge/Solvings/AbsoluteDifferenceChallenge_Kharchenko.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+final class AbsoluteDifferenceChallenge_Kharchenko: AbsoluteDifferenceChallengeProtocol {
+    func solve(_ n: Int) -> Int {
+        var result = Int()
+        
+        if n == 0 {
+            return 51
+        }
+        if n != 0 && n < 51 {
```

Comment:
Check abs method, and rethink your  decision. Your task is working, but abs can do it much easier